### PR TITLE
Update fusion toc and merge with master

### PR DESCRIPTION
**Pull request by georgechenchao**
1. Delete TOC.md in reference folder;
2. Delete TOC.md in conceptual folder;
3. Add TOC.yml in reference folder;
4. Add TOC.yml in conceptual folder;
5. Update configurations for using fusion TOC.

### CODE REVIEW

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[2017-03-09-profile/docs-ref-autogen/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-autogen/TOC.md) | :white_check_mark:Succeeded | |
[2017-03-09-profile/docs-ref-autogen/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-autogen/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/dla/account/compute-policy?view=azure-cli-2017-03-09-profile&branch=pr-en-us-451) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-conceptual/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-conceptual/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/dla/account/compute-policy?view=azure-cli-2017-03-09-profile&branch=pr-en-us-451) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-conceptual/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/dla/account/compute-policy?view=azure-cli-2017-03-09-profile&branch=pr-en-us-451) (azure-cli-2017-03-09-profile) |
[docfx.json](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/docfx.json) | :white_check_mark:Succeeded | |
[latest/docs-ref-autogen/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-autogen/TOC.md) | :white_check_mark:Succeeded | |
[latest/docs-ref-autogen/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-autogen/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-451) (azure-cli-latest) |
[latest/docs-ref-conceptual/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-conceptual/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-451) (azure-cli-latest) |
[latest/docs-ref-conceptual/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-451) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C8%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201708080907500842-451%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[2017-03-09-profile/docs-ref-autogen/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-autogen/TOC.md) | :white_check_mark:Succeeded | |
[2017-03-09-profile/docs-ref-autogen/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-autogen/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/dla/account/compute-policy?view=azure-cli-2017-03-09-profile&branch=pr-en-us-451) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-conceptual/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-conceptual/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/dla/account/compute-policy?view=azure-cli-2017-03-09-profile&branch=pr-en-us-451) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-conceptual/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/dla/account/compute-policy?view=azure-cli-2017-03-09-profile&branch=pr-en-us-451) (azure-cli-2017-03-09-profile) |
[appveyor.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/appveyor.yml) | :white_check_mark:Succeeded | |
[docfx.json](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/docfx.json) | :white_check_mark:Succeeded | |
[latest/docs-ref-autogen/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-autogen/TOC.md) | :white_check_mark:Succeeded | |
[latest/docs-ref-autogen/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-autogen/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-451) (azure-cli-latest) |
[latest/docs-ref-conceptual/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-conceptual/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-451) (azure-cli-latest) |
[latest/docs-ref-conceptual/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-451) (azure-cli-latest) |
[merge-toc.ps1](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/merge-toc.ps1) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C8%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201708080915042146-451%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: .openpublishing.publish.config.json [hunk 1]
```diff
@@ -40,7 +38,7 @@
     {
       "path_to_root": "_themes",
       "url": "https://github.com/Microsoft/templates.docs.msft",
-      "branch": "master",
+      "branch": "develop",
```
Comment (sptramer) [inline .openpublishing.publish.config.json]: Should published sites be going live off of the `develop` templates?

**Comment by rloutlaw** [comment]
Just a quick check in this PR-is the intended branch for the msft docs template supposed to be 'develop' and not 'master'? This is all under your control but it was something that was pointed out by the team.

**Comment by georgechenchao** [comment]
Yes. The build templates should be master, I will reopen this PR.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[2017-03-09-profile/docs-ref-autogen/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-autogen/TOC.md) | :white_check_mark:Succeeded | |
[2017-03-09-profile/docs-ref-autogen/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-autogen/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/dla/account/compute-policy?view=azure-cli-2017-03-09-profile&branch=pr-en-us-451) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-conceptual/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-conceptual/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/dla/account/compute-policy?view=azure-cli-2017-03-09-profile&branch=pr-en-us-451) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-conceptual/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/2017-03-09-profile/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/dla/account/compute-policy?view=azure-cli-2017-03-09-profile&branch=pr-en-us-451) (azure-cli-2017-03-09-profile) |
[appveyor.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/appveyor.yml) | :white_check_mark:Succeeded | |
[docfx.json](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/docfx.json) | :white_check_mark:Succeeded | |
[latest/docs-ref-autogen/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-autogen/TOC.md) | :white_check_mark:Succeeded | |
[latest/docs-ref-autogen/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-autogen/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-451) (azure-cli-latest) |
[latest/docs-ref-conceptual/TOC.md](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-conceptual/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-451) (azure-cli-latest) |
[latest/docs-ref-conceptual/TOC.yml](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/latest/docs-ref-conceptual/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/network/application-gateway/probe?view=azure-cli-latest&branch=pr-en-us-451) (azure-cli-latest) |
[merge-toc.ps1](https://github.com/Azure/azure-docs-cli-python/blob/masterfusiontoc/merge-toc.ps1) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C9%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201708090223303653-451%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**